content/en/synthetics/browser_tests/advance_options.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Advanced Options for Browser Test Actions
+kind: documentation
+description: Configure advance options for Browser Test Actions
+aliases:
+- /synthetics/browser_tests
+further_reading:
+- link: "synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn more about Browser Test Actions"
+---
+
+## Custom selector
+
+By default, [browser tests][1] use the Datadog locator system. It is recommended to only use custom selectors in edge cases as the Datadog locator system is what allows tests to be self-maintaining.
+
+Custom selectors are created by performing a step of interest in the recorder (**click**, **hover**, **assert**, etc.) on any element of your page. This specifies the kind of step that needs to be performed.
+
+To specify your custom locator:
+
+* Choose an [action][2] and complete this action in the Browser Test scenario panel.
+* Once the action is recorded in the left panel, click on the recorded action.
+* Click **Advanced options**.
+* The HTML element can then be selected either with an X-path or with a CSS class/ID, for example: `div`, `h1`, or `.hero-body`.
+* Once you have defined an element, click **Test** to highlight the element in the recording on the right.
+
+{{< img src="synthetics/browser_tests/advance_options/custom_selector.png" alt="Custom Selector" style="width:75%">}}
+
+By default the box **If user specified locator fails, fail test** is selected. This means that, by default, if the defined locator fails, the test will be considered a failure.
+
+You can decide to fallback on the regular Browser Test algorithm by deselecting the **If user specified locator fails, fail test** box.
+
+{{< img src="synthetics/browser_tests/advance_options/css.gif" alt="Fail Test Option">}}
```

Comment:
This gif pertains to the last bullet point of the list, do you think we could move that one up and move the img at the very bottom as it pertains to the `if user specified locator fails, fail test`?